### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

File: locale/id/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -400,7 +409,42 @@ msgstr "Tulis Kisahmu"
 
 #: gem/templates/yourwords/your_words_competition_tag.html:22
 msgid "Log in to Enter"
-msgstr "Log in untuk Masuk"
+msgstr "Log in untuk mengirim"
+
+#~ msgid "back to top"
```
Comment (saeedmb8) [inline locale/id/LC_MESSAGES/django.po]: @miltontony do you know why are these commented out?

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

File: locale/id/LC_MESSAGES/django.po (hunk 1 above)
Comment (miltontony) [inline locale/id/LC_MESSAGES/django.po]: It means they were removed a while back

**Comment by miltontony** [comment]
:+1:

**Pull request merged**